.travis.yml

```diff
@@ -47,8 +47,6 @@ before_install:
   - git submodule status
   - export DCTRL_ARG="--builddir=${DUNE_BUILD_DIR}  --opts=${OPTS}"
   # our local scripts look for an OPTS env entry
-  - ./local/bin/download_external_libraries.py
-  - ./local/bin/build_external_libraries.py
```

Comment:
We can just install the `eigen` debian packages instead, if we do not do so already...